### ISSUE TITLE
Cannot get user info even after 774

### ISSUE THREAD

**Issue by dvd42** [twintproject/twint#782]
# Issue Template
Please use this template!

>Make sure you've checked the following:

- [x] Python version is 3.6;
- [x] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [x] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
>twint -u "any_username"
### Description of Issue
>I can't get information for **any** user. It worked a few hours ago (with the patch #774 in place) but not it is not working. I tried changing my IP using a VPN but got the same results. Any help would be greatly appreciated.
 
### Environment Details
>Im using Linux and running twint inside pyenv from my terminal.

**Comment by mikaelyh** [comment]
wget
curl

all these tools doesn't work with Twitter since this last monday.

**Comment by calebglawson** [comment]
Twitter is responding with a blank webpage to the header sent in #774. Unfortunately, this is not fixed by removing the header, since Twitter then sends a page notifying the user that the browser is not supported.

Twitter's Legacy theme has been deprecated recently. Please see #756 for more info.

**Comment by mikaelyh** [comment]
> Twitter is responding with a blank webpage to the header sent in #774. Unfortunately, this is not fixed by removing the header, since Twitter then sends a page notifying the user that the browser is not supported.
> 
> Twitter's Legacy theme has been deprecated recently. Please see #756 for more info.

Thanks,

But I think, the problem is not only because of the deprecated legacy theme.

if I execute 
wget https://twitter.com/twitter  
with this UserAgent : 
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36

Twitter is responding a blankpage.

**Comment by Cheryl520** [comment]
Same problem again after #774, is there any solution now?

**Comment by yunusemrecatalcam** [comment]
For running Search and getting tweets, changing line 169 as
self.config.User_id = self.config.User_id_given#await get.User(url, self.config, self.conn, True)`
and setting id using tweepy api worked for me.
But as lookup uses actual result of get.User function I see that this workaround will not solve the issue for profile Lookup. Sad story
